packages/x-components/src/design-system/components/badge/default.scss

```diff
@@ -0,0 +1,28 @@
+.x-badge {
+  // border
+  border-radius: 50%;
```

Comment:
This border radius is going to be always round? Should be allow the option to change it?